### PR TITLE
Update project_gtd.py

### PR DESCRIPTION
**Pull request by ehirt**

### CODE REVIEW

**Comment by ehirt** [comment]
Pull request for issue #210 
Thanks Daniel for requesting as I did the pull request to my own branch instead of oca branch.

File: project_gtd/project_gtd.py [hunk 1]
```diff
@@ -102,8 +102,10 @@ def copy_data(self, cr, uid, id, default=None, context=None):
             context = {}
         if not default:
             default = {}
-        default['timebox_id'] = False
-        default['context_id'] = False
+        if not default.get('timebox_id', False):
```
Comment (dreispt) [inline project_gtd/project_gtd.py]: You can just use `if not default.get('timebox_id'): ` for the same effect.

**Review by dreispt — COMMENTED** [review]
It's not obvious to me - why is this fix needed?

**Comment by ehirt** [comment]
Hi Daniel,
thanks for the comments. I have streamlined the code accordingly.
The fix is required because the data_copy method does not follow the context and timesheet_id values provided in the default dictionary but overwrites them always with False. This is a reasonable default behaviour such that these data is not copied from the original task, but values for the new task shall be able to be provided through the default dictionary.
Hope the above description is argument enough for the fix

**Review by dreispt — APPROVED** [review]
OK, LGTM

**Comment by ehirt** [comment]
Hi Daniel,
How can I make these changes also available in branch 7, 8, 9?
do I have to patch the files per branch and then make individual pull requests?
Thanks
Etienne

**Comment by dreispt** [comment]
@ehirt Yes, we need one PR per branch.

**Comment by ehirt** [comment]
PR is not yet merged -> reopen

**Comment by ehirt** [comment]
Dear Daniel, 
Thanks again for reviewing and approving the PR. Can you also merge it?

**Comment by dreispt** [comment]
@ehirt I'm confused: this PR is for 10.0, but project_gtd hasn't been ported yet...

**Pull request merged**